### PR TITLE
port meta.yaml from netcdftime to cftime

### PR DESCRIPTION
**Pull request by jhamman**
rename `netcdftime` --> `cftime`

### CODE REVIEW

**Comment by jhamman** [comment]
see also https://github.com/Unidata/cftime/issues/36

**Comment by ocefpaf** [comment]
I'll archive the old feedstock.

File: recipes/cftime/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{% set version = "1.0.0a2" %}
+
+package:
+  name: cftime
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/c/cftime/cftime-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/cftime/meta.yaml]: This is not on PyPI yet, right?

Comment (jhamman) [reply recipes/cftime/meta.yaml]: Correct. I'll try to push it this evening.

**Comment by ocefpaf** [comment]
No problem. I'll restart the buildsame then.

**Comment by jhamman** [comment]
@ocefpaf - I think we're good now. Let's see if the builds pass.

**Comment by jhamman** [comment]
@ocefpaf - how long does it usually take for the feedstock to get built? We're not seeing it yet?

**Comment by ocefpaf** [comment]
The feedstock creation process is stalled until we releases conda-smithy 3, which should happen this week. Unfortunately this package got us right in the middle of the conda-build 3 transition. I'm working on a workaround to get this as a feedstock manually later today so we can move forward with the renaming before Monday.

**Pull request merged**